sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExec.scala

```diff
@@ -78,7 +77,6 @@ object GpuMetric extends Logging {
   val DESCRIPTION_GPU_OP_TIME = "GPU time"
   val DESCRIPTION_FETCH_TIME = "fetch time"
```

Comment:
Maybe we should take this opportunity to elaborate a bit on what 'fetch' means here?  Elsewhere in Spark, fetch implies shuffle, so it would be good to clarify.

Reply:
I updated this PR to remove `fetchTime` and replace it with `collectTime` (the opposite of what I was originally proposing). This is consistent with how Spark is using their `collectTime` metric.